lib/msal-browser/src/cache/BrowserStorage.ts

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ICacheStorage, Constants, PersistentCacheKeys, TemporaryCacheKeys } from "msal-common";
+import { CacheOptions } from "../app/Configuration";
+import { BrowserAuthError } from "../error/BrowserAuthError";
+import { BrowserConfigurationAuthError } from "../error/BrowserConfigurationAuthError";
+import { BrowserConstants } from "../utils/BrowserConstants";
+import { ErrorCacheKeys } from "msal-common/dist/utils/Constants";
+
+export class BrowserStorage implements ICacheStorage {
+
+    private cacheConfig: CacheOptions;
+    private windowStorage: Storage;
+
+    private clientId: string;
+    private rollbackEnabled: boolean;
+
+    constructor(clientId: string, cacheConfig: CacheOptions) {
+        if (!window) {
+            throw BrowserAuthError.createNoWindowObjectError();
+        }
+
+        const storageSupported = typeof window[cacheConfig.cacheLocation] !== "undefined" && window[cacheConfig.cacheLocation] !== null;
```

Comment:
Can be `!!window[cacheConfig.cacheLocation]`

Reply:
Done.